01-JS-DOM-APIs/resources/css/style.css

```diff
@@ -0,0 +1,12 @@
+section {
+  text-align: center;
+  visibility: hidden;
+}
+#slideSource {
```

Comment:
selectors should be writting in other way as we reviewed in the topic 0. For this case it should be `#slide-source`